src/Tokenizer/Analyzer/CommentsAnalyzer.php

```diff
@@ -190,6 +190,10 @@ private function isStructuralElement(Token $token)
      */
     private function isValidControl(Tokens $tokens, Token $docsToken, $controlIndex)
     {
+        if ($tokens[$controlIndex]->isGivenKind(T_RETURN) && false !== stripos($docsToken->getContent(), '@var')) {
```

Comment:
This condition will be true for any tag that contains `@var`, e.g. `@variable`. Please add such tag as a new test case to make sure it is still changed.